arednNetworkDesign/frequency_bands.rst

```diff
@@ -1,93 +1,92 @@
 
-Each band is divided into channels, each of which consists of a 5 MHz frequency offset identified by the center frequency of the channel and assigned a numerical label. In the example below you can see that a selected channel may use more or less of the frequency range based on the chosen channel width. The wider the channel, the more overlap there will be with adjacent channels. Wide channels have the effect of reducing the number of non-overlapping or non-interfering channels that will be available for use. When selecting channels and widths, be sure to use non-overlapping channels. Devices not using the same channel and channel width that overlap will interfere with one another and cannot communicate to coordinate the sharing of bandwidth.
+Cada banda se divide en canales, cada uno de los cuales consiste en un desplazamiento de frecuencia de 5 MHz identificado por la frecuencia central del canal y se le asigna una etiqueta numérica. En el siguiente ejemplo, puede ver que un canal seleccionado puede usar más o menos el rango de frecuencia en función del ancho del canal elegido. Cuanto más ancho sea el canal, más superposición habrá con los canales adyacentes. Los canales anchos tienen el efecto de reducir la cantidad de canales que no se superponen o que no interfieren y que estarán disponibles para su uso. Al seleccionar canales y anchos de banda, asegúrese de usar canales que no se superpongan. Los dispositivos que no usan el mismo canal y ancho de canal que se superponen interferirán entre sí y no pueden comunicarse para coordinar el uso compartido del ancho de banda.
 
 .. image:: _images/channel-width-example.png
    :alt: Channel Width Example
    :align: center
 
-All of the 33 cm band is shared with other FCC authorized users. All of the upper channels on the 13 cm band are shared with standard FCC Part 15 :abbr:`WiFi (IEEE 802.11x)` users, as are all of the lower channels on the 5 cm band. The only frequency range which amateur operators do not currently share with non-licensed users is the 9 cm band, in which the US military may occasionally operate radio location units. The following table lists each amateur radio band, frequency range, total bandwidth of allocation, and the number of channels that are available for AREDN |trade| networking.
+Toda la banda de 33 cm se comparte con otros usuarios autorizados de la FCC. Todos los canales superiores en la banda de 13 cm se comparten con los usuarios estándar de la FCC Parte 15: abbr: `WiFi (IEEE 802.11x)`, al igual que todos los canales inferiores en la banda de 5 cm. El único rango de frecuencia que los radioaficionados no comparten actualmente con usuarios sin licencia es la banda de 9 cm, en la cual el ejército de los EE. UU. puede operar ocasionalmente con radiobalizas. La siguiente tabla enumera cada banda de radioaficionados, rango de frecuencia, ancho de banda total de asignación y la cantidad de canales disponibles para AREDN | trade | redes.
```

Comment:
This actually wont apply to regions outside the US. In the EU (or at least spain) it may be such as:

Todos los canales superiores en la banda de 13 cm están también atribuido a aplicaciones ICM, por lo que se deben aceptar la interferencias perjudiciales provenientes de este uso.  Al igual que todos los canales superiores en la banda de 6 cm. Tanto las bandas de 35cm como en la de 9cm, no tenemos atribuciones especiales. Esto implica que se nos impondrían las mismas reglas propias de cada banda. 

But this is not legal nor technical advice.